luminance-webgl/src/webgl2/state.rs

```diff
@@ -140,6 +140,8 @@ impl WebGL2State {
     let face_culling_state = get_ctx_face_culling_state(&mut ctx);
     let face_culling_order = get_ctx_face_culling_order(&mut ctx)?;
     let face_culling_mode = get_ctx_face_culling_mode(&mut ctx)?;
+    load_webgl_extensions(&mut ctx)?;
```

Comment:
It feels odd to have this in the middle of the `let` declarations. Can you move it out? Either at top or bottom this section.

Reply:
Also, `webgl2`, please.

Reply:
I moved it to the top, in case an extension is ever needed for one of these gets.  Renamed to webgl2.